dracut.sh

```diff
@@ -2538,8 +2538,8 @@ if [[ $uefi == yes ]]; then
         fi
     fi
 
-    offs=$(objdump -h "$uefi_stub" 2> /dev/null | gawk 'NF==7 {size=strtonum("0x"$3);
-                offset=strtonum("0x"$4)} END {print size + offset}')
+    offs=$(($(objdump -h "$uefi_stub" 2> /dev/null | awk 'NF==7 {size=$3;
```

Comment:
What is the double `$(` shell execution used for?

Reply:
Arithmetic Expansion: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_0